modules/ROOT/pages/_partials/commons/common-releasenotes.adoc

```diff
@@ -45,21 +47,51 @@ endif::[]
 //
 // END RELEASE SELECTION BLOCK -- Decide which releases are relevant for a given platform
 
+// BEGIN RELEASE BLOCK -- Release 3-0-2 (All) -- August 2022
+// Clone this block for each maintenance release and edit release notes as required
+ifdef::param-3-0-5[]
+// Set release number
+:param-release-tag: -3-0-5
+
+[#maint-latest]
+[#maint-3-0-5]
+== 3.0.5 -- November 2022
+
+Version 3.0.5 of Couchbase Lite for {param-title} delivers a number of fixes and enhancements.
+
+=== Enhancements
+
+include::{module-partials}pn-issues-list.adoc[tag=Enhancements{param-release-tag}]
```

Comment:
I don't understand why some of the 3-0-5 are hard-coded and others are parametrized.
(Or how this works with later sections for other releases in the same document?)

But assume you've tested it and it looks ok...